huaweicloud/services/acceptance/cnad/data_source_huaweicloud_cnad_advanced_instances_test.go

```diff
@@ -0,0 +1,80 @@
+package cnad
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccDatasourceInstances_basic(t *testing.T) {
+	rName := "data.huaweicloud_cnad_advanced_instances.test"
+	dc := acceptance.InitDataSourceCheck(rName)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
```

Comment:
it's better to add an environment `HW_CNAD_INSTANCE_ID` to check the acceptance.

Reply:
OK, the environment is `HW_CNAD_ENABLE_FLAG`, use for skip cnad acceptance